content/cucumber/step-argument-transforms.md

```diff
@@ -7,7 +7,9 @@ title: Parameters in Step Definitions
 
 > TODO: Ruby specific feature. generalize.
 
-Step argument transforms help your step definitions be more by DRY by allowing you to refactor common operations that you perform on step definition arguments. Before each match captured by a step definition is yielded as an argument to the step definition block, an attempt is made to match them against registered `Transform` objects. If one of those captured matches match the regular expression of one of a `Transform` objects, its original string value is replaced with the result of what the `Transform` block yields.
+Step argument Transforms keep your Step Definitions DRY. They allow you to refactor common operations that you perform on Step Definition arguments. 
```

Comment:
Capital A on argument in "Step argument Transforms"?

Reply:
Not as sure about that one.  Can you tell me your reasoning?  

Anyone else have thoughts?

Reply:
My reasoning was "Step Argument" / "Step Argument Transform" like "Step Definitions".